pkg/console/install_panels.go

```diff
@@ -863,15 +863,10 @@ func addAskCreatePanel(c *Console) error {
 				Value: config.ModeJoin,
 				Text:  "Join an existing Harvester cluster",
 			},
-		}
-		installed, err := harvesterInstalled()
-		if err != nil {
-			logrus.Error(err)
-		} else if installed {
-			options = append(options, widgets.Option{
+			{
 				Value: config.ModeUpgrade,
 				Text:  "Upgrade Harvester",
```

Comment:
This value and text should be deleted too, as we don't want there to be an "Upgrade Harvester" item in this menu - the `harvesterInstalled()` function previously always returned `false`, so the `if installed` condition above would always be `false`, meaning this "Upgrade Harvester" item was never added before.